vehicle/tests/golden/compile/test-mnist/test-mnist.vcl

```diff
@@ -0,0 +1,46 @@
+--------------------------------------------------------------------------------
+-- Inputs and outputs
+
+-- Define the type for our input images
+--type Image = Tensor Rat [28, 28]
+
+--------------------------------------------------------------------------------
+-- Network
+
+-- Declare the network used to classify images. The output of the network is a
+-- score for each of the digits 0 to 9.
+@network
+mnist : Tensor Rat [28, 28] -> Vector Rat 10
+
+--------------------------------------------------------------------------------
+-- Definition of robustness around a point
+
+-- First we define the parameter `epsilon` that will represent the radius of the
+-- ball that we want the network to be robust in. Note that we declare this as
+-- a parameter which allows the value of `epsilon` to be specified at compile
+-- time rather than be fixed in the specification.
+
+--epsilon : Rat
+--epsilon = 0.01
```

Comment:
Please could you open issues for any bugs you've found in the translation of loss functions so we can track them?